src/components/ApplyGrid.tsx

```diff
@@ -29,14 +29,18 @@ const ApplyGrid: React.FC = () => {
 
         <ApplyCard
           title="Product Designer"
-          to="/apply/designer"
+          to="/apply/product_designer"
```

Comment:
how do we want to name our routes? is there a best practice? I feel like its usually product-designer rather than product_designer

Reply:
I'm fine with that too